README.md

```diff
@@ -1,4 +1,4 @@
-# PostWoman CLI  [![Postwoman](https://img.shields.io/badge/Made_for-Postwoman-hex_color_code?logo=Postwoman)](https://postwoman.io) [![Go Report Card](https://goreportcard.com/badge/github.com/athul/pwcli)](https://goreportcard.com/report/github.com/athul/pwcli)
+# hoppscotch CLI  [![hoppscotch](https://img.shields.io/badge/Made_for-Postwoman-hex_color_code?logo=Postwoman)](https://hoppscotch.io) [![Go Report Card](https://goreportcard.com/badge/github.com/athul/pwcli)](https://goreportcard.com/report/github.com/athul/pwcli)
```

Comment:
Don't you think the first letter should be capital. Could you change to Hoppscotch? And Do remove the "Made for PostWoman" badge, we'll add one once they update the badge